src/main/java/uk/gov/justice/digital/hmpps/keyworker/controllers/KeyworkerServiceController.java

```diff
@@ -248,4 +248,60 @@ public ResponseEntity allocate(
 
         return ResponseEntity.status(HttpStatus.CREATED).build();
     }
+
+    /* --------------------------------------------------------------------------------*/
+
+    @ApiOperation(
+            value = "Search for key workers within agency.",
+            notes = "Search for key workers using firstname or lastname",
+            nickname="keyworkersearch")
+
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "OK", response = KeyworkerDto.class, responseContainer = "List"),
+            @ApiResponse(code = 400, message = "Invalid request.", response = ErrorResponse.class, responseContainer = "List"),
+            @ApiResponse(code = 404, message = "Requested resource not found.", response = ErrorResponse.class, responseContainer = "List"),
+            @ApiResponse(code = 500, message = "Unrecoverable error occurred whilst processing request.", response = ErrorResponse.class, responseContainer = "List")  })
+
+    @GetMapping(path="/{agencyId}/members")
```

Comment:
I'm not happy with this endpoint name but would like to review when all endpoints in place